### PR TITLE
Glow Berries, when eaten give the player the Glowing effect for a short time

### LINKED ISSUE #45: Glow Berries, when eaten give the player the Glowing effect for a short time

### PR DESCRIPTION
**Pull request by michiel1106**
first pull request yay

### CODE REVIEW

**Comment by Flatkat** [comment]
please, for future PRs remember to stick to one PR per feature please. you can always make multiple branches on your fork for each pr.

**Comment by Flatkat** [comment]
also, "authors" section is for members of the team, please credit yourself on contributors instead

**Comment by michiel1106** [comment]
whoops yeah gonna do that

**Comment by Flatkat** [comment]
one last thing, don't reorganize the mixins on the list, they are meant to be in the same order feature appear on the feature tracker rather than alphabetical order.

**Comment by michiel1106** [comment]
alright, yeah I used the auto mixin add thingie, will remember to do it manually, should I just make a new pull request with the changes?

**Comment by Flatkat** [comment]
Also the description doesn't seem that fitting, since even if part of our work is QoL, we also rebalance and do other stuff. Will just replace it myself with the temporary short description we have for the modrinth page until someone writes a better one (I think velt was gonna do that)

File: src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(Item.class)
+public class GlowBerryMixin {
```
Comment (Flatkat) [inline src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java:19]: Add Feature description and credits as comments to the class. (use https://github.com/Team-Symphony/Harmony/blob/0.0.1-dev%2B1.21.0/src/main/java/dev/symphony/harmony/mixin/transportation/VehiclesMoveThroughLeaves.java as reference)

**Review by Flatkat — CHANGES_REQUESTED** [review]
Add Feature description and credits as comments to the class. (use https://github.com/Team-Symphony/Harmony/blob/0.0.1-dev%2B1.21.0/src/main/java/dev/symphony/harmony/mixin/transportation/VehiclesMoveThroughLeaves.java as reference)

File: src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java (hunk 1 above)
Comment (michiel1106) [reply src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java:19]: done

File: src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+// FEATURE: When Glowberries are eaten, they give the glowing effect for a configurable amount of time
```
Comment (Flatkat) [inline src/main/java/dev/symphony/harmony/mixin/food/GlowBerryMixin.java:16]: Tested the feature and it is ready to be merged!

**Review by Flatkat — APPROVED** [review]
Ready to merge!

**Pull request merged**